_layouts/doc.html

```diff
@@ -3,7 +3,7 @@
 <main class="doc">
   <article class="container">
     {% include ad.html %}
-    {{ content }}
+    {{ content | replace: '(recommended)', '<span aria-label="recommended" class="glyphicon glyphicon-ok"></span>' | replace: '(fixable)', '<span aria-label="fixable" class="glyphicon glyphicon-wrench"></span>' }}
```

Comment:
One thing we missed is that we should add the `title`1 property to the span. It helps the enduser to know what it is when he hovers over the icon.